tests/e2e/performance/io_workload/test_small_file_workload.py

```diff
@@ -500,13 +500,36 @@ def generate_kibana_link(self, index, columns):
         )
         return result
 
+    def collect_benchmark_logs(self):
+        """
+        Collecting the test log from all benchmark pods
+        """
+
+        # Getting full list of benchmark clients
+        self.full_client_list = get_pod_name_by_pattern(
+            self.client_pod_name, benchmark_operator.BMO_NAME
+        )
+
+        # Collecting logs from each pod
+        for clpod in self.full_client_list:
+            test_logs = self.pod_obj.exec_oc_cmd(f"logs {clpod}", out_yaml_format=False)
+            log_file_name = f"{self.full_log_path}/{clpod}-pod.log"
+            try:
+                with open(log_file_name, "w") as f:
+                    f.write(test_logs)
+                log.info(f"The Test log can be found at : {log_file_name}")
+            except Exception:
+                log.warning(f"Cannot write the log to the file {log_file_name}")
+        log.info("Logs from all client pods got successfully")
```

Comment:
Can we rephrase this to "Logs of all the client pods have been collected successfully"